TrackerApp/UI/Statistics/StatisticsViewController.swift

```diff
@@ -7,11 +7,163 @@
+        super.viewWillAppear(animated)
+        do {
+            records = try trackerRecordStore.fetchAllRecords()
+        } catch {
+            print(error.localizedDescription)
+        }
+        
+        tableView.reloadData()
+        checkRecords()
+    }
+    
+    //   MARK: - Methods
+    private func checkRecords() {
+        if records.count == 0 {
+            emptyStatisticsImageView.isHidden = false
+            emptyStatisticsLabel.isHidden = false
+            tableView.isHidden = true
+        } else {
+            emptyStatisticsImageView.isHidden = true
+            emptyStatisticsLabel.isHidden = true
+            tableView.isHidden = false
+        }
+    }
+    
+    private func setLayout() {
+        view.backgroundColor = .ypBackgroundScreen
+        
+        [titleLabel, emptyStatisticsLabel, emptyStatisticsImageView, tableView].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            view.addSubview($0)
+        }
+    }
+    
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            titleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            titleLabel.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 52),
+            
+            emptyStatisticsImageView.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            emptyStatisticsImageView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            
+            emptyStatisticsLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            emptyStatisticsLabel.topAnchor.constraint(equalTo: emptyStatisticsImageView.bottomAnchor, constant: 8),
+            
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            tableView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 77),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16),
+            tableView.heightAnchor.constraint(equalToConstant: 408)
+        ])
+    }
+}
+
+//MARK: - UITableViewDataSource, UITableViewDelegate
+extension StatisticsViewController: UITableViewDataSource, UITableViewDelegate {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        StatisticsCases.allCases.count
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        102
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: StatisticCell.identifier, for: indexPath) as? StatisticCell else { return UITableViewCell() }
+        
+        var title = ""
+        let subtitle = StatisticsCases.allCases[indexPath.row].name
+        
+        switch indexPath.row {
+        case 0:
+            title = String(0)
+            // TO DO
```

Comment:
```suggestion
            // TODO: - 
```
такой специальный вариант есть, как MARK: -

Reply:
Спасибо за подсказку)